assets/js/modules/analytics/datastore/properties.js

```diff
@@ -240,13 +243,14 @@ export const actions = {
 	 * @private
 	 *
 	 * @param {Array} properties Properties to add.
+	 * @param {Object} accountID Account ID to add.
 	 * @return {Object} Redux-style action.
 	 */
-	receiveProperties( properties ) {
+	receiveProperties( properties, { accountID } ) {
 		invariant( Array.isArray( properties ), 'properties must be an array.' );
```

Comment:
See other comment, let's also add an `invariant` call here to ensure `accountID` is passed.

Reply:
Fixed